source/agora/node/BlockStorage.d

```diff
@@ -937,10 +949,6 @@ public class MemBlockStorage : IBlockStorage
         // Basic consistenty checks
         assert(this.height_idx.length == this.hash_idx.length);
         assert(this.height_idx.length == this.blocks.length);
-
-        // Make sure we have no empty block
-        foreach (blk; this.blocks)
-            assert(blk.length > 0);
```

Comment:
Is this test failing now ?

Reply:
Ah I'll revert this, I initially replaced the serialized form with full blocks but that didn't work.